macros/edr/dbt_artifacts/upload_dbt_tests.sql

```diff
@@ -112,6 +112,7 @@
     {%- endif -%}
 
     {% set original_file_path = node_dict.get('original_file_path') %}
+    {# Becasue we use compiled sql for fetching data sampling, we currently not using the macro get_model_sql_from_node #}
```

Comment:
I think this comment is not relevant as the macro is called get_model_sql_from_node and here it's a test